docs/installation.rst

```diff
@@ -6,38 +6,47 @@ Installation
 
 There are different ways to install |project_name|:
 
-- **distribution package** - easy and fast if a package is available for your
-  Linux/BSD distribution.
-- **PyInstaller binary** - easy and fast, we provide a ready-to-use binary file
+- :ref:`distribution-package` - easy and fast if a package is
+  available from your operating system.
```

Comment:
"from your operating system" sounds a bit weird IMHO.

while a whole linux/bsd distribution could be considered as "operating system", it is not like that when you consider the OS X or Windows (Cygwin) way. Here the OS is rather separate from the distribution packages we mean.